debugger.h

```diff
@@ -12,7 +12,7 @@
 #ifndef _DEBUGGER_H
 #define _DEBUGGER_H
 
-#include <SDL.h>
+#include <SDL2/SDL.h>
```

Comment:
This change contradicts with the remark in the `FindSDL2.cmake` script:
```
# Note that the header path has changed from SDL2/SDL.h to just SDL.h
# This needed to change because "proper" SDL convention
# is #include "SDL.h", not <SDL2/SDL.h>. This is done for portability
# reasons because not all systems place things in SDL2/ (see FreeBSD).
```

Reply:
You're right. This is also noted [in the FAQ](https://wiki.libsdl.org/FAQDevelopment#Do_I_.23include_.3CSDL.h.3E_or_.3CSDL.2FSDL.h.3E.3F), so I'll change the includes to follow that.